### PR TITLE
session: fix global sysdate_is_now won't work

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/32784

Problem Summary:

### What is changed and how it works?

fix global sysdate_is_now won't work

### Check List

Tests 

- [x] Unit test

### Release note

```release-note
None
```

### CODE REVIEW

File: sessionctx/variable/sysvar.go [hunk 1]
```diff
@@ -1381,18 +1381,11 @@ var defaultSysVars = []*SysVar{
...
-	{Scope: ScopeGlobal | ScopeSession, Name: SysdateIsNow, Value: BoolToOnOff(DefSysdateIsNow), skipInit: true, Type: TypeBool,
+	{Scope: ScopeGlobal | ScopeSession, Name: SysdateIsNow, Value: BoolToOnOff(DefSysdateIsNow), Type: TypeBool,
 		SetSession: func(vars *SessionVars, s string) error {
 			vars.SysdateIsNow = TiDBOptOn(s)
 			return nil
 		},
-		GetGlobal: func(vars *SessionVars) (s string, err error) {
-			return strconv.FormatBool(GlobalSysdateIsNow.Load()), nil
-		},
-		SetGlobal: func(vars *SessionVars, s string) error {
```
Comment (JmPotato) [inline sessionctx/variable/sysvar.go:1392]: Why remove this function since it could be set globally?

Comment (Yisaer) [reply sessionctx/variable/sysvar.go:1392]: We don't need to inplement `SetGlobal` , `GetGlobal` here, the `setSysVariable` already handle it.

**Review by qw4990 — APPROVED** [review]

**Pull request merged**